nibabel/streamlines/__init__.py

```diff
@@ -0,0 +1,131 @@
+import os
+from ..externals.six import string_types
+
+from .header import Field
+from .compact_list import CompactList
+from .tractogram import Tractogram, LazyTractogram
+
+from nibabel.streamlines.trk import TrkFile
```

Comment:
All import seems relative, except this one.
I just changed it to test your version while keeping my vanilla nibabel, and it seems to works.